src/jbmc/jbmc_parse_options.cpp

```diff
@@ -640,6 +640,7 @@ void jbmc_parse_optionst::process_goto_function(
   const optionst &options)
 {
   symbol_tablet &symbol_table = function.get_symbol_table();
+  namespacet ns(symbol_table);
```

Comment:
Can this be moved closer to the function call where it is actually needed?

Reply:
I figured other passes are likely to want a namespace, so let's make it alongside the symtab it wraps?

Reply:
Can you maybe move that change to the previous commit even (where I have just made a suggestion of using `symbol_table`)?